Core/src/org/sleuthkit/autopsy/contentviewers/osaccount/OsAccountViewer.java

```diff
@@ -53,26 +53,29 @@ public OsAccountViewer() {
 
     @Override
     public void setNode(Node node) {
-        OsAccount osAccount = null;
+        Long osAccountId = null;
         try {
-            osAccount = node.getLookup().lookup(OsAccount.class);
-            if (osAccount == null) {
-                Optional<OsAccount> optional;
-                AbstractFile file = node.getLookup().lookup(AbstractFile.class);
-                if (file != null) {
-                    optional = file.getOsAccount();
-                    if (optional.isPresent()) {
-                        osAccount = optional.get();
-                    }
+            OsAccount osAccount = node.getLookup().lookup(OsAccount.class);
+            if (osAccount != null) {
+                dataPanel.setOsAccount(osAccount);
+                return;
+            }
+            
+            Optional<Long> optional;
+            AbstractFile file = node.getLookup().lookup(AbstractFile.class);
+            if (file != null) {
+                optional = file.getOsAccountObjectId();
+                if (optional.isPresent()) {
+                    osAccountId = optional.get();
                 }
             }
 
-            if (osAccount == null) {
+            if (osAccountId == null) {
                 DataArtifact dataArtifact = node.getLookup().lookup(DataArtifact.class);
                 if (dataArtifact != null) {
-                    Optional<OsAccount> optional = dataArtifact.getOsAccount();
+                    optional = dataArtifact.getOsAccountObjectId();
```

Comment:
I think this method went away or something.

Reply:
Oh nevermind the story that causes this story is still open.

Reply:
Yup, you need the PR for sleuthkit from the story.